CONTRIBUTING.md

```diff
@@ -31,6 +31,7 @@ Here are a few things you can do that will increase the likelihood of your pull
 
 If you are the current maintainer of this gem:
 
+1. Checkout to `main` and `git pull`
```

Comment:
We could also make the script do this part? Make sure it's on `main` and then `git pull` and confirm no conflicts?